app/src/main/java/com/github/se/cyrcle/ui/addParking/location/overlay/RectangleSelection.kt

```diff
@@ -74,15 +81,14 @@ fun RectangleSelection(
           if (hasDragged.value) {
             // Fill of the rectangle
             drawRect(
-                color =
-                    if (isAreaTooLarge) Red.copy(alpha = 0.5f)
-                    else Color(0xFF22799B).copy(alpha = 0.7f),
+                color = if (isAreaTooLarge) Red.copy(alpha = 0.5f) else Cerulean.copy(alpha = 0.7f),
                 topLeft = canvasCenter,
                 size = rectSize)
 
             // Outline of the rectangle
+            drawRect(color = Cerulean.copy(alpha = 0.7f), topLeft = canvasCenter, size = rectSize)
```

Comment:
Here you remove the if statement that displays the rectangle as red (with 50% opacity) if it is not valid

Reply:
A bad merge has occurred somewhere, I'll fix it ^^